include/targets/cuda/tunable_kernel.h

```diff
@@ -23,18 +23,18 @@ namespace quda
   */
   qudaError_t qudaLaunchKernel(const void *func, const TuneParam &tp, const qudaStream_t &stream, const void *arg);
 
-  /**
-     @brief This helper function indicates if the present
-     compilation unit has explicit constant memory usage enabled.
-  */
-  static bool use_constant_memory()
-  {
-#ifdef QUDA_USE_CONSTANT_MEMORY
-    return true;
-#else
-    return false;
-#endif
-  }
+//   /**
```

Comment:
I guess delete this code block rather than comment it.

Reply:
[3a4f46f](https://github.com/lattice/quda/pull/1569/commits/3a4f46f29c5000376812d5c0b6dd30817687d2fa)